integration_tests/mockApis/responses/prisonerBasicDetails.json

```diff
@@ -0,0 +1,7 @@
+{
+  "prisonerNumber": "ABC123",
```

Comment:
A12345 crops up a fair bit too - I suppose these should have the right format or could confuse people in future